ui/src/components/Markdown/index.jsx

```diff
@@ -199,17 +199,8 @@ export default class Markdown extends Component {
 
   render() {
     const { classes, children, className, ...props } = this.props;
-
-    /* eslint-disable react/no-danger */
     return (
-      <span
-        className={classNames(classes.root, className)}
-        dangerouslySetInnerHTML={{
-          __html: markdown.render(children),
-        }}
-        {...props}
-      />
+      <span className={classNames(classes.root, className)} {...props}>{markdown.render(children)}</span>
```

Comment:
This will cause all Markdown to be rendered as onscreen HTML.  I don't think that's what you want :)

I believe that the output of `markdown.render` is considered safe.

Reply:
Same as above!